src/osd/OSDMap.h

```diff
@@ -699,7 +699,7 @@ class OSDMap {
 
   void get_all_osds(set<int32_t>& ls) const;
   void get_up_osds(set<int32_t>& ls) const;
-  void get_out_osds(set<int32_t>& ls) const;
+  void get_out_existing_osds(std::set<int32_t>& ls) const;
```

Comment:
@badone Should we keep uniform set<int32_t> instead of std::set<int32_t> ?

Reply:
@pdvian Yes. I would keep it uniform even though the fully qualified form is more desirable. If you want to create a seperate PR to qualify the std::set references that would be cool. Check out the PR that fully qualified these in master and let me know if I can help.

Reply:
@badone the commit-id a192c1a525c3a4f0aa50e1ff282b028fbbb0b14c has above fully qualified form for these functions. The PR#27255 is quite big and need separate backport PR though we donot have tracker for 27255. Let me know your thoughts.

Reply:
@badone is out, let's go ahead without the `std::` for this PR.

Reply:
Sure. Let me do the changes accordingly.